Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDocumentLayout.h

```diff
@@ -52,6 +52,11 @@ class PDFDocumentLayout {
         TwoUpContinuous,
     };
 
+    enum class ShouldUpdateAutoSizeScale : bool {
+        Yes,
```

Comment:
*please* always use 0 == No for bool enums

Reply:
+1

Reply:
Also optional nit, but bool enums can be single-line

```cpp
enum class ShouldUpdateAutoSizeScale : bool { No, Yes };
```